modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -0,0 +1,77 @@
+= Point-in-Time Restore
+
+Point-in-Time Restore, introduced in version 4.1.0, enables users to restore the database to a previous time point using previously made backups, even though no backup was conducted at exactly that time point.
+This feature is based on xref:tigergraph-server:backup-and-restore:differential-backups.adoc[Differential Backup].
+
+== Usage
+
+To perform point-in-time restore, run the following command.
+The timepoint should be in timestamp format.
+
+[console, gsql]
+----
+$ gadmin backup restore <timepoint_to_restore> --time-point
+----
+
+For example, if you want to restore the cluster to the timepoint `2024-06-01T14:24:31Z` , run `gadmin backup restore 2024-06-01T14:24:31Z --time-point`. remember, the timepoint is precise to only seconds, other minor time parts will be truncated.
+
+The point-in-time restore is based on full backup and differential backup. So we need to ensure the time point to restore is not under these cases:
+
+
+* the time points before the first full backup or after the current time.
```

Comment:
```suggestion
* Time Points Before the First Full Backup or After the Current Time:
- The selected time point cannot be earlier than the first full backup or later than the current system time. This is because the restore process needs a reference point from an existing full backup, and restoring to a future time is not possible.
- Example: If the first full backup was made on 2024-06-01T00:00:00Z, you cannot restore to any time before this, such as 2024-05-31T23:59:59Z. Similarly, if the current time is 2024-06-15T12:00:00Z, you cannot restore to a time like 2024-06-16T00:00:00Z.
```